
in directory:
/ws/oi-userland/components/scientific/openblas/build/sparcv9/ctest

without this patch the build stops in the ctest folder:

OPENBLAS_NUM_THREADS=2 ./xdcblat1
 Real CBLAS Test Program Results
...

 Test of subprogram number  7         CBLAS_DNRM2    

Program received signal SIGBUS: Access to an undefined portion of a memory object.

Backtrace for this error:
/bin/sh: 186674: Abort(coredump)
make[1]: *** [Makefile:101: all1] Abort
make[1]: *** Waiting for unfinished jobs....

reported upstream with: https://github.com/OpenMathLib/OpenBLAS/issues/5552

--- OpenBLAS-0.3.30/ctest/Makefile.orig	2025-06-19 11:45:39.000000000 +0200
+++ OpenBLAS-0.3.30/ctest/Makefile	2025-11-27 11:00:35.850227063 +0100
@@ -69,9 +69,6 @@
 ifeq ($(BUILD_SINGLE),1)
 all1targets += xscblat1
 endif
-ifeq ($(BUILD_DOUBLE),1)
-all1targets += xdcblat1
-endif
 ifeq ($(BUILD_COMPLEX),1)
 all1targets += xccblat1
 endif
@@ -86,9 +83,6 @@
 ifeq ($(BUILD_SINGLE),1)
 	OMP_NUM_THREADS=2 ./xscblat1
 endif
-ifeq ($(BUILD_DOUBLE),1)
-	OMP_NUM_THREADS=2 ./xdcblat1
-endif	
 ifeq ($(BUILD_COMPLEX),1)
 	OMP_NUM_THREADS=2 ./xccblat1
 endif
@@ -99,9 +93,6 @@
 ifeq ($(BUILD_SINGLE),1)
 	OPENBLAS_NUM_THREADS=2 ./xscblat1
 endif
-ifeq ($(BUILD_DOUBLE),1)
-	OPENBLAS_NUM_THREADS=2 ./xdcblat1
-endif
 ifeq ($(BUILD_COMPLEX),1)
 	OPENBLAS_NUM_THREADS=2 ./xccblat1
 endif
@@ -273,15 +264,11 @@
 ifeq ($(BUILD_DOUBLE),1)
 # Double real
 ifeq ($(NOFORTRAN), $(filter 0 2,$(NOFORTRAN)))
-xdcblat1: $(dtestl1o) c_dblat1.o $(TOPDIR)/$(LIBNAME)
-	$(FC) $(FLDFLAGS) -o xdcblat1 c_dblat1.o $(dtestl1o) $(LIB) $(EXTRALIB) $(CEXTRALIB)
 xdcblat2: $(dtestl2o) c_dblat2.o $(TOPDIR)/$(LIBNAME)
 	$(FC) $(FLDFLAGS) -o xdcblat2 c_dblat2.o $(dtestl2o) $(LIB) $(EXTRALIB) $(CEXTRALIB)
 xdcblat3: $(dtestl3o) c_dblat3.o $(TOPDIR)/$(LIBNAME)
 	$(FC) $(FLDFLAGS) -o xdcblat3 c_dblat3.o $(dtestl3o) $(LIB) $(EXTRALIB) $(CEXTRALIB)
 else
-xdcblat1: $(dtestl1o) c_dblat1c.o $(TOPDIR)/$(LIBNAME)
-	$(CC) $(CFLAGS) -o xdcblat1 c_dblat1c.o $(dtestl1o) $(LIB) $(CEXTRALIB) $(filter-out -lgfortran,$(EXTRALIB))
 xdcblat2: $(dtestl2o) c_dblat2c.o $(TOPDIR)/$(LIBNAME)
 	$(CC) $(CFLAGS) -o xdcblat2 c_dblat2c.o $(dtestl2o) $(LIB) $(CEXTRALIB) $(filter-out -lgfortran,$(EXTRALIB))
 xdcblat3: $(dtestl3o) c_dblat3c.o $(TOPDIR)/$(LIBNAME)
